src/app/components/charts/workload-profile/workload-profile-radar-chart.component.scss

```
@@ -0,0 +1,33 @@
+  max-width: 100%;
+  display: flex;
+  justify-content: center;
+}
+
+.workload-profile-radar-chart__canvas--compare {
+  height: 460px;
+}
+
+.workload-profile-radar-chart__canvas--details {
+  height: 420px;
+  // max-height: 420px;
```

Stylelint (17.13.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)